Engine/Helper.cs

```diff
@@ -1323,156 +1323,168 @@ internal List<RuleSuppression> GetSuppressionsConfiguration(ConfigurationDefinit
         /// Suppress the rules from the diagnostic records list.
         /// Returns a list of suppressed records as well as the ones that are not suppressed
         /// </summary>
-        /// <param name="ruleSuppressions"></param>
-        /// <param name="diagnostics"></param>
```

Comment:
Shouldn't this contain the list of parameters here? I'm not sure why they're being deleted.

Reply:
Added more comments now. Originally took them out since they weren't filled in